packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md

```diff
@@ -24,13 +34,19 @@ The following environment variables can be set to configure the Proxy sample ins
 | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
 | `PROXY_BUNDLE_PATH`                        | Path to the JSS SPA app's `server.bundle.js`. Default can be seen in [config.js](./config.js) file.                                                                                                                    |
 | `PROXY_PORT`                              | Optional. Port which will be used when start sample. Default can be seen in [config.js](./config.js) file.                                                             |
-| `DEBUG`                  | Optional. Debug level for the proxy. Set the DEBUG environment variable to 'sitecore-jss:*,proxy*,http-proxy-middleware*' to see all logs.                                                                                                                  |
+| `DEBUG`                  | Optional. Debug level for the proxy. Set the DEBUG environment variable to 'sitecore-jss:*,proxy*,http-proxy-middleware*' to see all logs.
+
+## Pre-requisites
+
+1. SPA sample supports XM Cloud out of the box.
```

Comment:
You can add a mention of angular SPA here. Basically the only pre-requisite is to have angular SPA be built - and the build instructions are on the angular app side.

Reply:
I think we should not specify name of the particular framework in the proxy app since it will be used for other frameworks too. Repeating the build prerequisite doesn't hurt.